conf/global.properties

```diff
@@ -74,30 +74,30 @@ regex.id=[0-9\-]+
 #regex.id.nonqualified=[0-9\-]+
 #regex.id.qualified=[0-9A-Za-z\-]+:[0-9\-]+
 zimbraMessageIdDedupeCacheSize.default=3000
-AD.url=10.139.244.162
+AD.url=129.154.238.213
 AD.port=3268
-AD.domain=zmexch.eng.vmware.com
-AD.userName=administrator
-AD.password=z1mbr4Migration
-AD.passwordChangeCase=z1mbr4MigratiON
-AD.LDAPSearchBase=cn=users,dc=zmexch,dc=eng,dc=vmware,dc=com
+AD.domain=eng.zimbradev.com
```

Comment:
Should it be eng.zimbra.com or zimbradev.com instead of eng.zimbradev.com?

Reply:
@nilamraut24 New Ad domain is "eng.zimbradev.com" , so it is correctly set there.

Reply:
Ok @pallavikhairnar21 got it. Thank you.